grass7/gui/wxpython/wx.metadata/m.csw.update/m.csw.update.html

```diff
@@ -0,0 +1,86 @@
+<h2>DESCRIPTION</h2>
+
+<em>m.csw.update</em> update csw connections resource xml file (required
+by the <em>g.gui.cswbrowser</em> and <em>g.gui.metadata</em> module).
+If <b>-p</b> flag is used only printing instead of writing
+is done.
+<p>
+The module also allows validate conections resources xml file agains
+xsd schema, remove not valid and not active csw connnections resources
+candicates.
+
+<h2>NOTES</h2>
+
+For dependencies and installation instructions see
+<a href="http://grasswiki.osgeo.org/wiki/ISO/INSPIRE_Metadata_Support">
+wiki page</a>.
+
+<p>
+Writed new connections resources candidates are only active and valid.
+
+<h3>Writing new connections resources candicates</h3>
+
+Default source of the new candicates is spreadsheets file (*.ods), which
+is located in the module <i>config/</i> directory. It is possible to use
+the updated document directly from the web address, see <b>-l</b> flag.
+
+<h2>EXAMPLES</h2>
+
+Write new resources connections candicates (default):
+
+<div class="code"><pre>
+m.csw.update url=API-cases.ods
+</pre></div>
+
+<p>
+Write new resources connections candicates with printing summary info:
+
+<div class="code"><pre>
+m.csw.update -s
+</pre></div>
+
+<p>
+Print only all new connections resources candicates
+(with following format <i>'{Country}, {Govermental level}, {Api provider}: {Url}'</i>)
+with summary info:
+
+<div class="code"><pre>
+m.csw.update -ps
+</pre></div>
+
+<p>
+Print only active new connections resources candicates with summary info:
+
+<div class="code"><pre>
+m.csw.update -pas
+</pre></div>
+
+<p>
+Validate default connnections resources xml file against xsd schema plus
+validate individual csw connection resources (remove and print non active
+csw):
+
+<div class="code"><pre>
+m.csw.update -xc
+</pre></div>
+
+
+<h2>SEE ALSO</h2>
+
+<em>
+<a href="g.gui.cswbrowser.html">g.gui.cswbrowser</a>
+<a href="g.gui.metadata.html">g.gui.metadata</a>
```

Comment:
```suggestion
<em>m.csw.update</em> updates the CSW connections resource XML file (required
by the <em>g.gui.cswbrowser</em> and <em>g.gui.metadata</em> modules).
If the <b>-p</b> flag is used only printing instead of writing is done.
<p>
The module also allows validate the connections resources XML file against
XSD schema, remove invalid and not active CSW connections resources
candidates.

<h2>NOTES</h2>

For dependencies and installation instructions see the
<a href="http://grasswiki.osgeo.org/wiki/ISO/INSPIRE_Metadata_Support">
wiki page</a>.

<p>
Stored new connections resources candidates are only those being active and valid.

<h3>Writing new connections resources candidates</h3>

Default source of the new candidates is a spreadsheet file (*.ods), which
is to be stored in the module's <i>config/</i> directory. It is possible to use
the updated document directly from the web address, see <b>-l</b> flag.

<h2>EXAMPLES</h2>

Import and store new resources connections candidates (default):

<div class="code"><pre>
m.csw.update url=API-cases.ods
</pre></div>

<p>
Store new resources connections candidates along with printing a summary info:

<div class="code"><pre>
m.csw.update -s
</pre></div>

<p>
Print only all new connections resources candidates
(with following format <i>'{Country}, {Governmental level}, {Api provider}: {Url}'</i>)
with summary info:

<div class="code"><pre>
m.csw.update -ps
</pre></div>

<p>
Print only active new connections resources candidates with summary info:

<div class="code"><pre>
m.csw.update -pas
</pre></div>

<p>
Validate the default connections resources XML file against XSD schema plus
validate individual CSW connection resources (remove and print non active
CSWs):

<div class="code"><pre>
m.csw.update -xc
</pre></div>


<h2>SEE ALSO</h2>

<em>
<a href="g.gui.cswbrowser.html">g.gui.cswbrowser</a>,
<a href="g.gui.metadata.html">g.gui.metadata</a>
```

Reply:
(minor English review)

Reply:
Thanks, I already fixed it.